embedingModels/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "embeding_models"
+version = "0.1.0"
+description = ""
+authors = ["Kleczyk <mailnasyff@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+transformers = "^4.39.3"
+torch = "^2.2.2"
```

Comment:
**nitpick:** use `~major.minor` instead of `^major.minor.patch`, it'll be "safer" as some tools can introduce breaking changes with minors (but they shouldn't)